lib/actions/fcs.py

```diff
@@ -42,20 +42,14 @@ def _check_args(self, req, supported_default, supported):
 
     def _corpora_info(self, value, max_items):
         resources = []
+        corpora_list = [value]
         if value == 'root':
-            corpora_list = plugins.get('corparch').get_all(self._session_get('user', 'id'))
-            if "sort_corplist" in corpora_list:
-                corpora_list = corpora_list["sort_corplist"]
-            else:
-                corpora_list = corpora_list["corplist"]
-        else:
-            corpora_list = [{"ident": value}]
+            corpora_list = plugins.get('auth').permitted_corpora(self._session_get('user'))
 
-        for i, corpora_d in enumerate(corpora_list):
+        for i, corpus_id in enumerate(corpora_list):
```

Comment:
What does enumerate do to dicts? permitted_corpora returns a dict https://github.com/ufal/lindat-kontext/blob/kontext-dev/lib/plugins/lindat_auth/__init__.py#L109

Reply:
Iterating over a Python dict means to iterate over its keys exactly the same way as with dict.keys()

Reply:
fixed it in https://github.com/ufal/lindat-kontext/pull/141/commits/60c5d4f6af56dae89702c0d43f9547a3f36c7694
(the other PR that is branched of this one)